fsd_utils/authentication/config.py

```diff
@@ -9,6 +9,7 @@
 import enum
 
 config_var_auth_host = "AUTHENTICATOR_HOST"
+config_var_logout_url_override = "LOGOUT_URL_OVERRIDE"
```

Comment:
Would prefer all capitals for constants, but understand you've just kept the existing convention so no need to worry about it